src/js/tech/ChromecastTech.js

```diff
@@ -144,6 +152,45 @@ ChromecastTech = {
       this._playSource(source, 0);
    },
 
+   /**
+    * generate track
+    * @param trackData
+    */
+   generateTrack: function(trackData, id) {
+      var sub = new chrome.cast.media.Track(id, chrome.cast.media.TrackType.TEXT);
+
+      sub.trackContentId = trackData.src;
+      sub.trackContentType = 'text/vtt';
+      sub.subtype = chrome.cast.media.TextTrackType.SUBTITLES;
+      sub.name = trackData.language;
+      sub.language = trackData.language;
+      return sub;
+   },
+
+   /**
+    * _onChangeTrack
+    * @private
+    */
+   _onChangeTrack: function() {
+      var castSession = cast.framework.CastContext.getInstance().getCurrentSession(),
+          media = castSession.getMediaSession(),
+          index, subtitles, tracksInfoRequest, i;
+
+
+      if (castSession) {
+         index = [];
+         subtitles = this.videojsPlayer.remoteTextTracks();
+         for (i = 0; i < subtitles.length; i++) {
+            if (subtitles[i].mode === 'showing') {
+               index = [ i ];
```

Comment:
It's a style thing I know, but is feels like it should be this?
```suggestion
               index = [i];
```

Reply:
No, eslint configs wants it with spaces.